portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,94 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
+    <meta charset="utf-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <script src="https://kit.fontawesome.com/0d47b72ee3.js" crossorigin="anonymous"></script>
   </head>
+
   <body>
+    <nav class="navbar">      
+      <div class="logo">
+        <h3><a href="#">Charles Ogbogu</a></h3>
+      </div>
+      <ul class="nav-links" id="js-menu">
+        <li><a href="index.html" class="navbar-link">About</a></li>
+        <li><a href="resume.html" class="navbar-link">Resume</a></li>
+        <li><a href="#" class="navbar-link">Experience</a></li>
+        <li><a href="#" class="navbar-link">Sport</a></li>
+        <li><a href="#footer" class="navbar-link">Contact</a></li>
+      </ul>
+
+      <span class="navbar-toggle" id="js-navbar-toggle">
+        <i class="fas fa-bars"></i>
+      </span> 
+
+      <script src="https://code.jquery.com/jquery-3.5.1.js"></script>
+      <script>
+        $(document).ready(function() {
+          $('.navbar-toggle').click(function() {
+            $('ul').toggleClass('active');
+          })
+        })
+      </script>
```

Comment:
Script stuff like this tends to be at the bottom just before </body> tag. The <script src="script.js"></script> can also be there. Scripts originally would get tossed at the bottom for performance since HTML/JS is one thread. While it's loading jquery, the page doesn't keep rendering. So putting the scripts at the bottom allows the page to keep rendering and load JS files later.